docs/Release_Notes.gfs.v16.3.11.md

```diff
@@ -4,7 +4,17 @@ GFS V16.3.11 RELEASE NOTES
 PRELUDE
 -------
 
-Update to the WAFS package.
+At GFS v16.3 implementation, UPP and WAFS package were updated to be ready to produce high resolution WAFS output to meet 2023 ICAO milestone with one switch. The switch was turned off at GFS v16.3. Now that UKMO has started to produce their corresponding high resolution WAFS output, EMC will work with NCO to turn on the switch.
+
+Additionally, EMC is updating WAFS ecf definition file to 1)trigger WAFS blending job 5 min later 
+from 4:25 to 4:30 to compensate for 5-10 min delay in receiving UKMO high resolution data;
+2)stop producing blended 1.25 deg WAFS file per ICAO milestone. 
+Both sides agreed that we sill stop waiting for data from the other side at T+4:45.
+
+Finally, NESDIS informed EMC that they will implement new global satellite composite data in Jan 2024. 
+This data set is used by one of WAFS packages. NESDIS provided a sample data set which Yali used to 
+develop an updated config file that works with current ops and to-be-implemented satellite data. 
+This updated config file will be implemented too.
```

Comment:
Following the meeting discussion on Nov 3, the NESDIS changes are not going to be included in this release.